hazelcast/src/main/java/com/hazelcast/internal/metrics/impl/MetricsUtil.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.metrics.impl;
+
+import com.hazelcast.internal.metrics.MetricTarget;
+import com.hazelcast.internal.metrics.ProbeLevel;
+
+import java.util.Collection;
+import java.util.Set;
+
+import static com.hazelcast.internal.metrics.MetricTarget.ALL_TARGETS;
+import static com.hazelcast.internal.metrics.MetricTarget.ALL_TARGETS_BUT_DIAGNOSTICS;
+import static com.hazelcast.internal.metrics.MetricTarget.DIAGNOSTICS;
+import static com.hazelcast.internal.metrics.MetricTarget.asSet;
+import static com.hazelcast.internal.metrics.ProbeLevel.DEBUG;
+import static java.util.Collections.emptySet;
+
+final class MetricsUtil {
+    private MetricsUtil() {
+    }
+
+    static Collection<MetricTarget> excludedTargets(Collection<MetricTarget> exclusions, ProbeLevel level) {
```

Comment:
It would be nice to add javadocs for methods in this utility class and, maybe, to rename this method. Current name doesn't give any hints on differences with `#getExcludedTargets`.

Reply:
Sure, I will do this when finalizing the PR :+1: